src/classifier/Options.h

```diff
@@ -56,22 +62,35 @@ class Options {
   virtual void Initialize();
 
   // Get option values.
-  const string &GetTrainingFilePath() { return file_train_; };
-  const string &GetTestFilePath() { return file_test_; };
-  const string &GetModelFilePath() { return file_model_; };
-  const string &GetOutputFilePath() { return file_prediction_; };
-  int GetNumEpochs() { return train_epochs_; };
-  double GetRegularizationConstant() { return train_regularization_constant_; }
-  const string &GetTrainingAlgorithm() { return train_algorithm_; }
-  double GetInitialLearningRate() { return train_initial_learning_rate_; }
+  const string &GetTrainingFilePath() { 
```

Comment:
Don't use this indentation. Either 

int Foo() { return 0; }

of 

int Foo() {
  return 0;
}

but never

int Foo() {
  return 0; }

Also, if it fits in one line (with <= 80 chars) I prefer the former solution.